### PR TITLE
Reacthome

### PR DESCRIPTION
**Pull request by vc0622**
Updated the orders page UI so it looks a lot cleaner.
It only displays the necessary things like Customer , Date, Picked_UP, and Pack.

### CODE REVIEW

File: specx_fulfillment/spec/spec_helper.rb [hunk 1]
```diff
@@ -1,5 +1,4 @@
-
-#################################################################
+#
```
Comment (m35) [inline specx_fulfillment/spec/spec_helper.rb]: These should be #####... lines

File: specx_fulfillment/spec/spec_helper.rb [hunk 2]
```diff
@@ -8,8 +7,7 @@
 require 'codeclimate-test-reporter'
 CodeClimate::TestReporter.configuration.git_dir = '..'
 CodeClimate::TestReporter.start
-#################################################################
-
+#
```
Comment (m35) [inline specx_fulfillment/spec/spec_helper.rb]: These should be ##### lines

Comment (xiaodi528) [inline specx_fulfillment/spec/spec_helper.rb]: changed it

File: specx_fulfillment/spec/spec_helper.rb (hunk 1 above)
Comment (xiaodi528) [inline specx_fulfillment/spec/spec_helper.rb]: changed it

**Pull request merged**